packages/onboard-react/src/components/WalletDialog/WalletDialog.tsx

```diff
@@ -1,48 +1,109 @@
-import React, { FC, useCallback, useMemo, useState } from "react";
+import React, { FC, useCallback, useEffect, useMemo, useState } from "react";
+
 import {
   Dialog,
   VerticalTabs,
   VerticalTabsItem,
+  VerticalTabsMobile,
   Button,
   ButtonStyle,
   Heading,
   DialogSize,
 } from "@sovryn/ui";
 import { connectWallet$ } from "@sovryn/onboard-core/dist/streams";
+import resolveConfig from "tailwindcss/resolveConfig";
+import tailwindConfig from "@sovryn/tailwindcss-config";
+
 import { FilterType, WalletList } from "../WalletList/WalletList";
 import { InstructionsTab } from "../InstructionsTab/InstructionsTab";
 import { HardwareWallets } from "../HardwareWallets/HardwareWallets";
 import styles from "./WalletDialog.module.css";
+import { ButtonBack } from "../ButtonBack/ButtonBack";
+import { useSubscription } from "../../hooks/useSubscription";
+import { selectAccounts$ } from "@sovryn/onboard-hw-common";
+
+const config = resolveConfig(tailwindConfig);
 
 type WalletDialogProps = {
   isOpen: boolean;
 };
 
 const WalletDialog: FC<WalletDialogProps> = ({ isOpen }) => {
+  const { inProgress } = useSubscription(selectAccounts$);
+  const [isMobile, setIsMobile] = useState(false);
   const [index, setIndex] = useState(0);
+  const [indexMobile, setIndexMobile] = useState<null | number>(null);
```

Comment:
Why do we have separate indexes for mobile and desktop? I think we should use the same and reset it in case platform changes.

Reply:
@tiltom so basically we have an active index by default for the desktop version, which starts from 0, and for the mobile version, we do not have an active index by default. I can try to make it work with one index, but it requires additional changes in the packages/ui library for that.

Reply:
That makes sense, you don't need to adjust it.